charts/akeyless-api-gateway/templates/cache.yaml

```diff
@@ -72,6 +72,10 @@ spec:
         env:
         - name: REDIS_DISABLE_COMMANDS
           value: "FLUSHDB,FLUSHALL"
+        - name: REDIS_MAXMEMORY_POLICY
```

Comment:
please add resources to the redis pod..

Reply:
Added resources:

```yaml
cache:
  image:
    repository: docker.io/bitnami/redis
    tag: 6.2
    pullPolicy: Always
  resources:
  # limits:
  #   cpu: 500m
  #   memory: 2Gi
  # requests:
  #   cpu: 300m
  #   memory: 1Gi
```

Reply:
https://github.com/akeylesslabs/helm-charts/pull/251/files#diff-847c215886e59b4f9922d3a1938ed28722b71ffd9894dd0524458f6995e6ac02R25

Reply:
Done.